packages/node_modules/@ciscospark/react-component-title-bar/src/index.js

```diff
@@ -23,7 +22,7 @@ function TitleBar({avatarId, children, name}) {
   return (
     <div className={classNames('ciscospark-title-bar', styles.titleBar)}>
       <div className={classNames('ciscospark-avatar-container', styles.avatarContainer)}>
-        <PresenceAvatar avatarId={avatarId} iconSize={'30px'} name={name} />
+        <PresenceAvatar avatarId={avatarId} iconSize="30px" name={name} />
```

Comment:
The react rule doesn't require these strings any more?

Reply:
Yea. That was an oversight on my part. I think we should enable it as a warn, but as is we tend to ignore the rule and get around it anyways. What do you think? Is there another way we can encourage or enforce not using raw strings when it’s displayed text?

Reply:
I haven't ever gotten around it, but I do like the idea of it. Although now that I'm reading the rule: https://github.com/yannickcr/eslint-plugin-react/blob/HEAD/docs/rules/jsx-no-literals.md I think this is just for string within a component and not params.